GoogleForCreators/web-stories-wp
# UI Simplification: Add Design panel button to replace background media

## Context



## Summary
Refactors code to have a separate Upload button component.
Uses this to allow replacing background media from the Page background panel directly.

- [x] Track button usage



## Relevant Technical Choices



## To-do



## User-facing changes
| Before | After |
| ------ | ----- |
| <img width="299" alt="Screenshot 2022-01-10 at 19 51 47" src="https://user-images.githubusercontent.com/3294597/148814349-a637b711-9d1a-4fcc-b146-a94a2738d4fa.png"> | <img width="298" alt="Screenshot 2022-01-10 at 19 48 53" src="https://user-images.githubusercontent.com/3294597/148813877-ae992309-bb52-4e25-aa5a-01e8dba51eee.png"> |


## Testing Instructions






This PR can be tested by following these steps:

1. Add background media
2. Click "Replace" button in the Page background panel
3. Verify it's possible to both choose media from the library / upload new media
4. Verify the "Upload" button in the Media tab still works as expected.


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
Adds new event `replace_background_media`.


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #10108
